### ISSUE TITLE
Leading zeros for decimal point Warning matching decimals greater than 9

### ISSUE THREAD

**Issue by mrosata** [rossPatton/stylint#242]
This should only show on numbers such as `0.1` and `0.125` not for `60.5%` or `10.25rem`. I'm using SublimeLinter which in turn uses the npm module `stylint`.

```bash
SublimeLinter: stylint: mobile-mini.styl ['/home/michael/.nvm/versions/node/v5.2.0/bin/stylint', '@'] 
SublimeLinter: stylint output:
Warning: leading zeros for decimal points are unecessary
File: /tmp/SublimeLinter3-michael/mobile-mini.styl
Line: 5: max-width: 60.5%
```

If this wasn't the place to post this I appologize. I did goto the SublimeLinter-Stylint repo first but after looking through their code a bit it seems that all the output is piped from this module. Thanks!

**Comment by rossPatton** [comment]
should be an easy one to fix

**Comment by nicgordon** [comment]
Nice work @maranomynet. I'd like to see this fix merged in.

**Comment by maranomynet** [comment]
Actually, looking at the source and tests, while fixing this bug (and a couple of other PRs), it seems like big parts of this linter need a complete rewrite/redesign. Some bits seem to only work by pure luck. :-/
